libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -32,6 +33,17 @@ const dashboardTable = [
     },
 ];
 
+const assertTableValues = ["Direct Sales", "Inside Sales"];
```

Comment:
what is different between 2 tables? as I see in graphene it uses table with title `true` and table with title `2021`